### DISCUSSION TITLE
Sort/Filter Entries by 'Entry Type'

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9192] [Ideas] [answered]
> Created by: Dan (dan@switch.je) on 2016/01/21 17:34:21 +0000
> Votes at time of UserVoice import: 5

-----------------------------------

Would be nice if you could sort/filter entries by entry type when viewing them under 'Entries'. If you are looking for a specific type of entry it can be a bit tedious scrolling through hundreds of entries looking for that specific type.

**Comment by angrybrad** [comment]
> Posted by Ryan Masuga (ryan@gomasuga.com) on 2016/09/16 16:15:29 +0000
-----------------------------------

Can&#39;t believe this isn&#39;t voted up more. Not being able to filter by entry type, coupled with the infinite scrolling on Edit screens is less than useful at trying to find entries of a type.

**Comment by KatieMFritz** [comment]
YES

**Comment by brandonkelly** [comment]
@KatieMFritz putting you down for a maybe

**Comment by ccsantos23** [comment]
Would be nice if this plugin was built-in:  https://plugins.craftcms.com/sidebarentrytypes

**Comment by aaronbushnell** [comment]
Was hoping this would be one of the options you could sort by in Craft 3.2 with the sorting changes but it doesn't appear that's the case. Would love to see this in a future update!

**Comment by mihow** [comment]
Is this on the roadmap for a future version?

**Comment by brandonkelly** [comment]
Yep, we will be making this possible in Craft 4.

**Reply by alexmillman2394** [reply]
what about craft 3? Pretty please?

**Reply by brandonkelly** [reply]
You can do it with a plugin: https://plugins.craftcms.com/cpfilters

**Comment by brandonkelly** [comment, marked as answer]
This has been implemented for Craft 4 🎉  (#10075)

**Reply by brandonkelly** [reply]
Just added the ability to sort by entry type as well, for Craft 4.1 (d349a076d64e0419a3114e04f4b3de87b598dbf9).

**Reply by KatieMFritz** [reply]
YAYYYYYYYY

**Reply by brandonkelly** [reply]
Craft 4.1 is out with this ✨
